src/remediations/remediations.format.js

```diff
@@ -373,3 +373,15 @@ exports.issueSystems = function (issue, total) {
         data: formatted
     };
 };
+
+exports.planNames = function (names, total, limit, offset, sort, system) {
+    return {
+        meta: {
+            count: 1,
```

Comment:
Since the `data:` we're returning is an array, `meta.count` should be the number of items in the array (`names.length`) and `meta.total` would be the total number of items.

Reply:
Oh right I didn't mean to leave it like this! My bad